libkbfs/folder_branch_ops.go

```diff
@@ -2239,7 +2238,20 @@ func (fbo *folderBranchOps) finalizeMDRekeyWriteLocked(ctx context.Context,
 		}
 		mdOps = jServer.delegateMDOps
 	}
-	mdID, err := mdOps.Put(ctx, md)
+
+	var key kbfscrypto.VerifyingKey
```

Comment:
Maybe the comment describing why we have to pass through a verifying key needs to go here along with the ticket number.